snapcraft/_store.py

```diff
@@ -281,34 +274,27 @@ def create_key(name):
         # yet.
         enabled_names = set()
     if name in enabled_names:
-        raise RuntimeError(
-            'You have already registered a key named "{}".'.format(name))
+        raise errors.KeyAlreadyRegisteredError(name)
     subprocess.check_call(['snap', 'create-key', name])
 
 
 def _maybe_prompt_for_key(name):
     keys = list(_get_usable_keys(name=name))
     if not keys:
         if name is not None:
-            raise RuntimeError(
-                'You have no usable key named "{}".\nSee the keys available '
-                'in your system with `snapcraft keys`.'.format(name))
+            raise storeapi.errors.NoSuchKeyError(name)
```

Comment:
why storeapi.errors and in other just errors? Maybe consolidate with storeapi.errors